### DISCUSSION TITLE
"Could not find `visit` in `syn`" in IntellJ.

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#5819] [Work In Progress - GUI] [answered]
Somehow, I cannot avoid this error when running clippy globally:
```
error[E0432]: unresolved import `syn::visit`
  --> lib/rust/macro-utils/src/lib.rs:18:10
   |
18 | use syn::visit::Visit;
   |          ^^^^^ could not find `visit` in `syn`

```

It does not come from my code. No amount of git cleaning seem to work. The `./run lint` command from console does work.

Anyone has any hints?

**Comment by farmaazon** [comment, marked as answer]
Huh, no wonder, we missed the feature here:
```
diff --git a/Cargo.toml b/Cargo.toml
index 96d44128a..9d41e1d49 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -128,6 +128,7 @@ syn = { version = "1.0", features = [
   "extra-traits",
   "printing",
   "parsing",
+  "visit",
   "visit-mut",
 ] }
 quote = { version = "1.0.23" }
```
With that change, it works. I will make a PR for that.

Seemingly, some WASM dependencies turn on this feature, so we hadn't to do that on WASM builds.

**Reply by wdanilo** [reply]
Strange!
